docs/guide/piece-pages.md

```diff
@@ -1,257 +1,162 @@
 
-```js
-// modules/page/index.js
-// Add the new page type to the "Type" dropdown for new pages
+## The index page template
 
-module.exports = {
-  options: {
-    types: [
-      {
-        name: 'default-page',
-        label: 'Default'
-      },
-      {
-        name: 'product-page',
-        label: 'Product Index'
-      },
-      {
-        name: '@apostrophecms/home-page',
-        label: 'Home'
-      }
-    ]
-  }
-};
-```
-
-```js
-// modules/product-page/index.js
-// Configure our new piece page type
-module.exports = {
-  extend: '@apostrophecms/piece-page-type',
-  options: {
-    label: 'Product Index Page'
-  }
-};
-```
+Index page templates look very similar to other page templates. Look for the new features in this example blog index page.
 
 ```django
-{# modules/product-page/views/index.html #}
+{# modules/article-page/views/index.html #}
 
-{% import '@apostrophecms/pager:macros.html' as pager with context %}
 {% extends "layout.html" %}
+{% import '@apostrophecms/pager:macros.html' as pager with context %}
 
 {% block main %}
-  {% for product in data.pieces %}
-    <h2>
-      <a href="{{ product._url }}">{{ product.title }}: {{ product.price }}</a>
-    </h2>
-    <section>{% area product, 'description' %}</section>
+  <h1>{{ data.page.title }}</h1>
+
+  {% for article in data.pieces %}
+    <article>
+      <h2>
+        <a href="{{ article._url }}">{{ article.title }}</a>
+      </h2>
+    </article>
   {% endfor %}
 
-  {# The pager macro now takes a `class` option to set `class` attributes. #}
   {{ pager.render({
     page: data.currentPage,
     total: data.totalPages,
-    class: 'my-pager-class'
+    class: 'blog-pager'
   }, data.url) }}
 {% endblock %}
 ```
 
-```django
-{# modules/product-page/views/show.html #}
-{% extends "layout.html" %}
-{% set product = data.piece %}
+### `data.pieces` and other unique `data` properties
 
-{% block main %}
-  {# The layout already output the title for us #}
-  <h4>Price: {{ product.price }}</h4>
-  <section>{% area product, 'description' %}</section>
-{% endblock %}
+The first new thing there is the `import` statement, but we'll get back to that. Let's talk about the **loop over `data.pieces`**.
+
+```django
+{% for article in data.pieces %}
+  <article>
+    <h2>
+      <a href="{{ article._url }}">{{ article.title }}</a>
+    </h2>
+  </article>
+{% endfor %}
 ```
 
-That's all we need to create a basic paginated index page for all of our products, with "virtual" subpages for the individual products, based on the `slug` field of each piece.
+In addition to standard information, index page templates have access to `data.pieces`, which is an array of piece document objects. They will contain the title and URL as shown here, but also contain other piece data. This lets you do things like adding thumbnail images to listed items.
```

Comment:
I feel like there has to be a quicker way to say "you can access the piece schema data with `data.pieces`.